perf/src/main.rs

```diff
@@ -0,0 +1,118 @@
+use std::net::{IpAddr, Ipv6Addr, SocketAddr};
+
+use anyhow::{Context, Result};
+use futures::StreamExt;
+use tracing::{debug, error, info};
+
+#[tokio::main]
+async fn main() {
```

Comment:
This always starts the server. I think quinn should also contain a client. Should this be an extra executable? Or maybe make it possible to configure via CLI (`perf --client`, `perf --listen ip`) which peer is used.